src/components/Header.jsx

```diff
@@ -1,5 +1,30 @@
 import { Link } from 'react-router-dom'
+import { userStore } from '../store/userStore'
+import userIcon from '/user.svg'
+import './Header.css'
 
 export const Header = () => {
-	return <header>{<Link to="/">HOME</Link>}</header>
+	const isLoggedIn = userStore.getState().isLoggedIn
```

Comment:
I see you are using the getState() method. :)
Do you think we should use one of Zustand's state access methods?

I've read that using the useStore hook
const isLoggedIn = useStore((state) => state.isLoggedIn);

is better because it helps keep your code responsive, concise, and consistent.

We can discuss this later  when we meet :)